### PR TITLE
Added Delete User Operation

### PR DESCRIPTION
**Pull request by mehulmathur16**
/claim #591

### CODE REVIEW

**Comment by maxgurewitz** [comment]
Will take a look shortly!

File: packages/dashboard/src/pages/users/[id].page.tsx [hunk 1]
```diff
@@ -119,6 +122,46 @@ const User: NextPage<UserPageProps> = function User(props) {
...
+    }
+
+    apiRequestHandlerFactory({
+      request: userDeleteRequest,
+      setRequest: setUserDeleteRequest,
+      responseSchema: EmptyResponse,
+      onSuccessNotice: `Deleted User`,
+      onFailureNoticeHandler: () =>
+        `API Error: Failed to delete User`,
+      setResponse: () => {},
```
Comment (maxgurewitz) [inline packages/dashboard/src/pages/users/[id].page.tsx:150]: Here, can you redirect the user back to the `/users` page.?

**Review by maxgurewitz — CHANGES_REQUESTED** [review]
Small change requested!

File: packages/dashboard/src/pages/users/[id].page.tsx (hunk 1 above)
Comment (mehulmathur16) [reply packages/dashboard/src/pages/users/[id].page.tsx]: Sure @maxgurewitz , this is done.

**Review by maxgurewitz — APPROVED** [review]

**Pull request merged**